core/tests/puppeteer-acceptance-tests/functions/is-element-clickable.ts

```diff
@@ -18,8 +18,14 @@
 
 export default function isElementClickable(element: Element): boolean {
   /**
-   * This function gets the the overlapping element if any by checking the
+   * This function gets the overlapping element if any by checking the
    * element that is present in the center of the target element's position.
+   *
+   * @param {Element} element The target element to check for overlapping
+   * elements.
+   * @param {ShadowRoot | Document} parent The parent element to check for
```

Comment:
Sorry, still don't understand this. I think the terminology is confusing here -- the parent element is not the immediate parent of the target element in the HTML tree, right? It's more like the root/document element -- if so, would rootElement be a better name?

Similar notes for the "parent" in other docstrings.

Reply:
Done